SQL/0000-00-00-schema.sql

```diff
@@ -1157,7 +1157,7 @@ CREATE TABLE `family` (
 
 CREATE TABLE `examiners` (
   `examinerID` int(10) unsigned NOT NULL auto_increment,
-  `full_name` varchar(255) default NULL,
+  `full_name` varchar(255) CHARACTER SET utf8 COLLATE utf8_bin default NULL,
```

Comment:
char set should be utf8mb4

Reply:
If case sensitivity is the issue, I think utf8mb4 is case insensitive.

Reply:
Some case sensitive collation now exist but maybe not for the version we use and maybe not with the proper ordering.